src/rules/no-extraneous-dependencies.js

```diff
@@ -277,6 +292,7 @@ module.exports = {
           packageDir: { type: ['string', 'array'] },
           includeInternal: { type: ['boolean'] },
           includeTypes: { type: ['boolean'] },
+          exclude: { type: ['string', 'array'] },
```

Comment:
for the array, let's ensure it's unique, and that the items can only be of type string